frontend/src/app/state/exercise/exercise.selectors.ts

```diff
@@ -62,7 +62,8 @@ export const selectTransferPoints = (state: AppState) =>
 
 export const selectTileMapProperties = (state: AppState) =>
     state.exercise.tileMapProperties;
-
+export const selectPretriageEnabledConfiguration = (state: AppState) =>
+    state.exercise.configuration.pretriageEnabled;
```

Comment:
maybe just one selector for the whole configuration.  `selectConfiguration`
And you can then use the property you want like this `configuration.pretriageEnabled`